libsrc/c64/soft80_conio.s

```diff
@@ -0,0 +1,230 @@
+;
+; Low level stuff for screen output/console input
+;
+
+        .constructor    soft80_init, 24
```

Comment:
I see. Then I suggest to just drop the `, 24` meaning the prio defaults to 7. This is the prio used be CONIO constructors on the Apple II and NES - which seems to be fine.

Does the PCE CONIO contructor prio 24 happen to be also just copy past?